packages/utils/src/conversions.ts

```diff
@@ -220,3 +220,49 @@ export function bufferToHexadecimal(value: Buffer | Uint8Array): string {
 
     return Buffer.from(value).toString("hex")
 }
+
+/**
+ * Converts a buffer to a base64 string. It accepts 'Buffer' or 'Uint8Array'.
+ * @param value The buffer to convert.
+ * @returns The converted base64 string.
+ */
+export function bufferToBase64(value: Buffer | Uint8Array): string {
+    requireTypes(value, "value", ["Buffer", "Uint8Array"])
+
+    return Buffer.from(value).toString("base64")
+}
+
+/**
+ * Converts a base64 string to a buffer. This function does not check
```

Comment:
Can you document (and ideally also test in the utest) what happens if it's not?
A quick experiment in Node suggests that what happens is that invalid characters are ignored, rather than an error being thrown.

Reply:
I confirm it ignores invalid characters. The documentation also mentions it: https://nodejs.org/api/buffer.html#buffers-and-character-encodings. Checking if a base64 is valid is not trivial and I'd avoid that in this function. I'll add more documentation.

Reply:
I think that's fine so long as the behavior is documented.

A regex can check validity of Base64 pretty reasonably, but that's non-trivial at the level of cost of Base64 conversion, yes.  For PODs I used a regex at the level of key/signature decoding, when I know the specific size and other properties I'm looking for.